assets/css/styles.css

```diff
@@ -1,10 +1,17 @@
+@import url(//db.onlinewebfonts.com/c/d2c52a64cb362f280d891dd4980160e5?family=Edo);
```

Comment:
No need of adding these codes

/* Edo Font */

@import url('http://fonts.cdnfonts.com/css/edo');

Just add this code only @bhavy2202